subekashi/templates/subekashi/base/header.html

```diff
@@ -1,15 +1,27 @@
 {% load static %}
-<header>
-    <div class="header-left">
-        <a href="{% url 'subekashi:top' %}"><img name="thumbnail" class="logo" src="{% static 'subekashi/image/icon.png' %}" alt="全て歌詞の所為です。のロゴ"></a>
-        <a href="{% url 'subekashi:top' %}" class="logomoji">全て歌詞の所為です。</a>
+<header id="header">
```

Comment:
`#header`は使われていないので不要です。
```suggestion
<header>
```